source/ed/build_helper.cpp

```diff
@@ -426,12 +452,25 @@ VJ builder::vj(const std::string& line_name,
                const std::string& uri,
                const std::string& meta_vj,
                const std::string& physical_mode){
-    return vj_with_network("base_network", line_name, validity_pattern, block_id,
+    return vj_with_network("base_network", line_name, "", validity_pattern, block_id,
+                           wheelchair_boarding, uri, meta_vj, physical_mode);
+}
+
+VJ builder::vj_with_route(const std::string& line_name,
```

Comment:
Thanks for the tip :+1:
It's easier to use and I don't have to modify any unit tests to make it works anymore.